framework/src/controller/index.ts

```diff
@@ -12,5 +12,6 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-export { Controller } from './controller';
-export { InMemoryChannel, IPCChannel } from './channels';
+export * from './controller';
+export * from './channels';
+export * from './event';
```

Comment:
If we are not using other exports then we can avoid exposing them and especially the users will not use controller related stuff for their app as most of it is very internal to framework. We can revert this change